changelog.dd

```diff
@@ -227,6 +229,27 @@ assert([4, 7, 5].enumerate.maxElement!`a.value` == tuple(1, 7));
 -------
 )
 
+$(LI $(LNAME2 quantize, Added `std.math.quantize`.)
+    $(P $(XREF math, quantize) rounds to the nearest multiple of some number.
+        Features:)
+    $(UL
+        $(LI The rounding method can be specified using the `rfunc` parameter. By
+            default, the current rounding mode will be used, which is typically
+            "banker's rounding".)
```

Comment:
@andralex 
> Also, does the notion of "current" rounding mode preclude purity of this function?

For better or worse, access to the floating-point status and control flags is permitted as a deliberate hole in D's `pure`, like memory allocation. `rint()` is marked as `pure`. Regardless, the purity of `quantize()` itself is simply inferred depending on the `rfunc` parameter.

I don't think it would be too big of a deal to change the default `rfunc` to something else later, considering that D doesn't guarantee that the current rounding mode will *actually* be respected anywhere, anyway. (See the recent mega-thread about the handling of floating-point by the CTFE engine.)